crates/router/src/connector/shift4/transformers.rs

```diff
@@ -95,7 +101,7 @@ impl Shift4AuthorizePreprocessingCommon for types::PaymentsPreProcessingData {
 }
 #[derive(Debug, Serialize)]
 pub struct Shift4PaymentsRequest {
-    amount: String,
+    amount: MinorUnit,
```

Comment:
@swangi-kumari @deepanshu-iiitu should this be `StringMinorUnit`? can you please confirm this?

Reply:
Actually it needs to be integer only. I think our previous integration was not correct.
https://dev.shift4.com/docs/api